operations/enterprises.md

```diff
@@ -21,7 +36,17 @@ Returns all company profiles of the enterprise, possible filtered by their ident
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `Ids` | array of string | optional | If specified, returns only companies with the specified identifiers. |
+| `Ids` | array of string | optional | Unique identifiers of [Company](enterprises.md#company)s. |
+| `Names` | array of string | optional | Names of [Company](enterprises.md#company)s. |
+| `CreatedUtc` | [Time Filter](enterprises.md#time-filter) | optional | Interval of [Company](enterprises.md#company)s creation date and time. |
```

Comment:
I fixed in master https://github.com/MewsSystems/gitbook-connector-api/commit/2960aca80b2e79fe7168c206485b686a862b9f8c
Companys -> Companies